common/src/main/java/com/wynntils/features/MythicFoundFeature.java

```diff
@@ -52,11 +54,25 @@ public void onMythicFound(MythicFoundEvent event) {
             return;
         }
 
-        // Lootrun reward
+        // Lootrun rewards
         Optional<GearItem> gearItem = Models.Item.asWynnItem(itemStack, GearItem.class);
         if (gearItem.isPresent()) {
             sendLootrunDryStreakMessage(
                     StyledText.fromComponent(event.getMythicBoxItem().getHoverName()));
+            return;
+        }
+
+        Optional<InsulatorItem> insulatorItem = Models.Item.asWynnItem(itemStack, InsulatorItem.class);
+        if (insulatorItem.isPresent()) {
+            sendLootrunDryStreakMessage(
+                    StyledText.fromComponent(event.getMythicBoxItem().getHoverName()));
+            return;
+        }
+
+        Optional<SimulatorItem> simulatorItem = Models.Item.asWynnItem(itemStack, SimulatorItem.class);
+        if (simulatorItem.isPresent()) {
+            sendLootrunDryStreakMessage(
+                    StyledText.fromComponent(event.getMythicBoxItem().getHoverName()));
         }
```

Comment:
Is this the only place this check is used? Does the mythic found event fire for these items?

Reply:
Oops yeah that does need it too